src/pubsub/src/generated/gapic/.sidekick.toml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[general]
+specification-source = 'google/pubsub/v1'
+service-config = 'google/pubsub/v1/pubsub_v1.yaml'
+
+[codec]
+copyright-year = '2025'
```

Comment:
I think you want to use this:

https://github.com/googleapis/google-cloud-rust/blob/e0de9a07308f638eba00b86684e80f89b72ae7a2/src/storage/src/generated/gapic/.sidekick.toml#L45

That will stop the generator from emitting the Cargo.toml and README files.

Reply:
Q: do we want to use HTTP transport for the admin clients?